src/tabletraits.jl

```diff
@@ -1,63 +1,13 @@
-using TableTraits
-using TableTraitsUtils
+using Tables
 
-TableTraits.isiterable(x::NDSparse) = true
-TableTraits.isiterabletable(x::NDSparse) = true
+Tables.AccessStyle(::Type{<:NextTable}) = Tables.ColumnAccess()
+sch(x::Type{<:NamedTuple}) = x
+sch(x::Type{T}) where {T <: Tuple} = NamedTuple{(Symbol("_$i") for i = 1:length(T.parameters)), T}
+Tables.schema(x::NextTable) = sch(eltype(x))
+Base.getproperty(x::Tuple, i::Int) = getfield(x, i)
+Tables.columns(x::NextTable) = columns(x)
+Tables.rows(x::NextTable) = rows(x)
 
-function TableTraits.getiterator(source::S) where {S <: NDSparse}
-    return rows(source)
-end
+table(x; kwargs...) = table(Tables.columntable(x); copy=false, kwargs...)
```

Comment:
Maybe worth considering:

`Tables.columns(NamedTuple, x)` instead of a new `columntable`? `Tables.rows(Array, x)` would stand for `rowtable`. Mentioning the type seems to help explain what these methods do better than *table.

cc @andyferris

Reply:
Yeah, this is interesting. I'm not sure how I feel about `columntable` to be honest. Shashi's idea does seem to be extended, which is a big advantage.